### PR TITLE
Adjusted uploader to send files and show success message

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/37810)

Now that the BE is working on a endpoint for COE we need to add code to submit the files uploaded by the Veteran. This PR adds that code as well as code to show a success message when they are uploaded.

## Testing done
unit tests will be added in a subsequent PR

## Screenshots

Screen Shot 2022-03-01 at 9 01 14 AM

## Acceptance criteria
- [x] The standalone version accepts multiple documents (when connected to the BE)
- [ ] The form version no longer has a file type field
- [ ] A PR has been requested for merging

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx [hunk 1]
```diff
@@ -85,8 +86,14 @@ const DocumentUploader = () => {
...
-        <va-alert status="success" visible>
-          <h3 slot="headline">Your files have been uploaded</h3>
+        <va-alert
+          background-only
+          close-btn-aria-label="Close notification"
+          show-icon
+          status="success"
+          visible
+        >
+          <div>Your files have been uploaded</div>
```
Comment (timwright12) [inline src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:96]: Should probably be a `<p>`

**Review by timwright12 — APPROVED** [review]

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx [hunk 2]
```diff
@@ -1,131 +1,76 @@
...
+  };
 
-  const onTextInputValueChange = useCallback(e => {
-    dispatch({
-      type: DOC_DESC,
-      documentDescription: { dirty: true, value: e?.value },
+  const onTextInputValueChange = e => {
+    setState({
+      ...state,
+      documentDescriotion: e.value,
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:38]: *documentDiscription

Comment (jerekshoe) [inline src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:38]: *documentDiscription

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx (hunk 1 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:96]: @timwright12 I agree, I will update this. Interestingly I copied this code from the storybook on alerts so we might want to update the example code and when you change it to a `p` tag the padding is off from the styles inherited from paragraph tags on our site https://design.va.gov/storybook/?path=/docs/components-va-alert--background-only-with-icon 

Screen Shot 2022-03-10 at 10 13 37 AM

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx [hunk 3]
```diff
@@ -135,10 +80,25 @@ const DocumentUploader = () => {
...
+      )}
+      {state.successMessage ? (
+        <va-alert
+          background-only
+          close-btn-aria-label="Close notification"
+          show-icon
+          status="success"
+          visible
+        >
+          <div>Your files have been uploaded</div>
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:96]: Shouldn't the text be `Your documents were successfully uploaded.`? That's what I'm seeing in the prototype

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx [hunk 4]
```diff
@@ -169,19 +129,13 @@ const DocumentUploader = () => {
...
+        errorMessage={state.errorMessage}
       />
-      <button
-        className="vads-u-margin-top--3 usa-button"
-        onClick={onSubmit}
-        type="button"
-      >
-        Submit uploaded documents
-      </button>
+      <button onClick={onSubmit}>Submit files</button>
```
Comment (jerekshoe) [inline src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx:138]: We should add a `type` attribute to this button

**Review by timwright12 — APPROVED** [review]

**Review by jerekshoe — APPROVED** [review]
I would recheck the success alert text again, but other than that, LGTM

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx (hunk 2 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx]: @jerekshoe Man my spelling is awful. Updated this

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx (hunk 3 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx]: @jerekshoe Good catch, updated this

File: src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx (hunk 4 above)
Comment (cohnjesse) [reply src/applications/lgy/coe/status/components/DocumentUploader/DocumentUploader.jsx]: @jerekshoe Updated this

**Pull request merged**